src/ncp-spinel/SpinelNCPTaskGetNetworkTopology.cpp

```diff
@@ -333,6 +333,36 @@ SpinelNCPTaskGetNetworkTopology::TableEntry::get_as_string(void) const
 ValueMap
 SpinelNCPTaskGetNetworkTopology::TableEntry::get_as_valuemap(void) const
 {
-	// TODO: return the child info as a value map dictionary.
-	return ValueMap();
+    ValueMap entryMap;
+    uint64_t addr;
+    
+    addr  = (uint64_t) mExtAddress[7];
+    addr |= (uint64_t) mExtAddress[6] << 8;
+    addr |= (uint64_t) mExtAddress[5] << 16;
+    addr |= (uint64_t) mExtAddress[4] << 24;
+    addr |= (uint64_t) mExtAddress[3] << 32;
+    addr |= (uint64_t) mExtAddress[2] << 40;
+    addr |= (uint64_t) mExtAddress[1] << 48;
+    addr |= (uint64_t) mExtAddress[0] << 56;
+
+    entryMap.insert( std::pair<std::string, boost::any>( "extAddress",     addr               ));
+    entryMap.insert( std::pair<std::string, boost::any>( "rolc16",         mRloc16            ));
```

Comment:
I suggest that we define all these strings in a header file somewhere (maybe in wpan-properties), something like this: 
```
#define kWPANTUNDValueMapKey_ChildNeighborEntry_ExtAddress        "ExtAddress"
#define kWPANTUNDValueMapKey_ChildNeighborEntry_Rloc16            "RLOC16" 
#define kWPANTUNDValueMapKey_ChildNeighborEntry_LinkQualityIn     "LinkQualityIn"
...
```
This way there is one place they are defined and the same `#define`s can later be used in sapphire code.